client/hotrod-client/src/main/java/org/infinispan/client/hotrod/security/ElytronSaslProviders.java

```diff
@@ -0,0 +1,37 @@
+package org.infinispan.client.hotrod.security;
+
+import java.security.Provider;
+import java.util.Arrays;
+
+import org.infinispan.client.hotrod.logging.Log;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.1
+ **/
+public class ElytronSaslProviders {
+   /**
+    * Install the Elytron SASL security providers. Should be invoked before constructing an authentication-enabled
+    * RemoteCacheManager
+    */
+   public static void installSecurityProviders() {
```

Comment:
I don't understand where this method should be invoked. 
Is the user responsible to invoke it before creating the `RemoteCacheManager`?